test/unit/org/apache/cassandra/tools/AuditLogViewerTest.java

```diff
@@ -61,6 +73,75 @@ public void tearDown() throws IOException
         }
     }
 
+    @Test
+    public void testNoArgs()
+    {
+        try (ToolRunner tool = runner.invokeTool(toolPath))
```

Comment:
Atm {{ToolRunner}} is {{AutoClosable}} so it's good to close it in case in the future `close()` does sthg meaningful imo. Let's see how `ToolRunner` looks when we finish our discussions about it first, then we can revert/change this as it's a pain to change every time we go back/forth on it.